### PR TITLE
dumpling: process nan when chunk is zero

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/39082

Problem Summary:

### What is changed and how it works?

add a check for chunk is zero

### Check List

Tests 

- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: dumpling/export/status.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func (d *Dumper) GetStatus() *DumpStatus {
 	ret.EstimateTotalRows = ReadCounter(d.metrics.estimateTotalRowsCounter)
 	ret.CurrentSpeedBPS = d.speedRecorder.GetSpeed(ret.FinishedBytes)
 	if d.metrics.progressReady.Load() {
+		if d.metrics.totalChunks.Load() == 0 {
```
Comment (D3Hunter) [inline dumpling/export/status.go:74]: maybe add a comment on when `totalChunks`=0

**Review by D3Hunter — COMMENTED** [review]
rest lgtm

**Review by D3Hunter — APPROVED** [review]

**Review by buchuitoudegou — APPROVED** [review]

**Pull request merged**